pkg/cmdline/cmdline.go

```diff
@@ -161,6 +161,21 @@ func GetInitFlagMap() map[string]string {
 	return getFlagMap(initflags)
 }
 
+// GetFlagsForModule gets all flags for a designated module
+// and returns them as a space-seperated  string designed to be passed to insmod
+func GetFlagsForModule(name string) string {
```

Comment:
write a unit test for this?

Reply:
this could probably actually be

```go
func (c CmdLine) FlagsForModule(name string) string {

}

func FlagsForModule(name string) string {
  return NewCmdLine().FlagsForModule(name)
}
```

which makes it testable code.

Reply:
It seems like the other cmdline functions follow the pattern of running once.Do(cmdLineOpener) instead of calling the function with a cmdline. Would you rather that pattern? Will write a test either way

Reply:
I don't think it makes a difference, your choice

Reply:
Updated latest with unit tests